net/Socket.hpp

```diff
@@ -939,19 +939,22 @@ class SocketPoll
 
     void transferSocketTo(const std::weak_ptr<Socket>& socket,
                           const std::weak_ptr<SocketPoll>& toPoll,
-                          SocketDisposition::MoveFunction cb)
+                          SocketDisposition::MoveFunction toCb,
```

Comment:
These are a bit cryptically named I think. Can we go for wild long names cbAfterArrivalInNewPoll, and cbAfterInsertionIntoNewPoll with more documentation =) took me a while to unwind what's going on.